### PR TITLE
Visual fixes for metrics plots in experiment tracking

### PR DESCRIPTION
**Pull request by tynandebold**
Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Since we released `5.2.0` I've noticed some small visual inconsistencies (thanks to @AntonyMilneQB for pointing one of these out to me). This PR fixes those.

## Development notes

1. Ensure tooltip on metrics plots is in correct position so this doesn't happen:
    Screen Shot 2023-01-20 at 15 28 11
3. Update IconButton tooltip logic to precent the following from happening (reverting some of #1139).
    Screen Shot 2023-01-20 at 15 21 47
4. Improve warning message for empty plots in exp. tracking to add the warning to each run and prevent this:
    Screen Shot 2023-01-20 at 15 21 06

## QA notes

View the Gitpod link or checkout the branch and run Viz locally to see these three things no longer happen.

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js [hunk 1]
```diff
@@ -99,7 +100,7 @@ export const ParallelCoordinates = ({
 
   const handleMouseOverMetric = (e, key) => {
     const runsCount = graph.find((each) => each[0] === key)[1].length;
-    const { x, y, direction } = getTooltipPosition(e);
+    const { x, y, direction } = getTooltipPosition(e, sidebarVisible);
```
Comment (tynandebold) [inline src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js:103]: We use the `Boolean` of `sidebarVisible` to determine if we need to adjust the tooltip's position.

File: src/components/experiment-tracking/run-dataset/run-dataset.scss [hunk 2]
```diff
@@ -109,11 +109,11 @@
   width: 250px;
 }
 
-.details-dataset__tranistion-group-wrapper {
+.details-dataset__transition-group-wrapper {
```
Comment (tynandebold) [inline src/components/experiment-tracking/run-dataset/run-dataset.scss:112]: This was a typo. Fixed it in this PR.

**Review by rashidakanchwala — APPROVED** [review]
LGTM

**Review by AntonyMilneQB — APPROVED** [review]
Fixes the tooltip bug I spotted 👍

**Pull request merged**